index.json

```diff
@@ -1,7 +1,5 @@
 {
   "$schema": "http://json.schemastore.org/prettierrc",
-  "printWidth": 80,
-  "singleQuote": true,
-  "trailingComma": "es5",
-  "endOfLine": "lf"
-}
\ No newline at end of file
+  "printWidth": 120,
+  "singleQuote": true
```

Comment:
I think we should have a debate on this rule.
All plugins / autocompletes use double quotes and every time I see this I ask myself "Why did we choose single quotes?"

Reply:
Honestly I do not know, it think we got this habit since ever at simplifield. 

I would say that this is because ' is a shift less than a "

![image](https://user-images.githubusercontent.com/1308790/106958288-6fb71b80-6739-11eb-9f8d-697e4d1fdbe3.jpeg)

But as far as I am concerned I would not mind if prettier fix those for me, we could change that too.

Do you want me to add it to the adr and kill 2 birds with one stone ?